tasks/security_agent.py

```diff
@@ -10,19 +10,28 @@
 from subprocess import check_output
 
 from .utils import bin_name, get_gopath, get_build_flags, REPO_PATH, get_version, get_git_branch_name, get_go_version, get_git_commit, get_version_numeric_only
-from .build_tags import get_default_build_tags
+from .build_tags import get_build_tags, LINUX_ONLY_TAGS
+from .go import generate
 
 BIN_DIR = os.path.join(".", "bin", "security-agent")
 BIN_PATH = os.path.join(BIN_DIR, bin_name("security-agent", android=False))
 GIMME_ENV_VARS = ['GOROOT', 'PATH']
 
+DEFAULT_BUILD_TAGS = [
```

Comment:
Could we move this to task/build_tags.py under something like `DEFAULT_SECURITY_AGENT_BUILD_TAGS` ?

Reply:
Any reason not to keep these scoped to security_agent.py?

Reply:
it was meant to be the place to compute build tag. But I see the cluster agent is also separate. I think it's fine. We can merge everything if needed later.